Dockerfile

```diff
@@ -2,3 +2,7 @@ FROM nginx:latest
 
 RUN rm /etc/nginx/conf.d/default.conf
 COPY ./conf /etc/nginx/conf.d
+RUN apt-get update && apt-get install -y git && \
```

Comment:
It's too vague for me & could be hard to maintain (imagine that this `RUN` is cached by docker).